tests/test_bouquets/test_bouquets.py

```diff
@@ -55,3 +56,18 @@ def test_bouquet_500(self):
 
         self.assertEqual(response.status_code, 500)
         self.assertEqual(re['error'], 'failed to add bouquet')
+        
+    def test_delete_bouquet(self):
+        response = self.app_test.delete("/v2/bouquets?bouquet_id=100")
+        self.assertTrue(b"The bouquet you want to delete is not found" in response.data)
+        self.assertEqual(response.status_code, 404)
+
+    def test_delete_bouquet_correctly(self):
+        response = self.app_test.delete("/v2/bouquets?bouquet_id=1")
```

Comment:
What makes the bouquet with id 1 pass and the one with 100 fail?

Reply:
In the test, we have 1 bouquet already saved in the database(id=1). The bouquet with id 100 does not exist. We can not delete a bouquet that does not exist.

Reply:
Alright